### PR TITLE
Create functionality

### PR DESCRIPTION
**Pull request by don-yoon**

### CODE REVIEW

File: src/pages/CarIndex.js [hunk 1]
```diff
@@ -3,19 +3,19 @@ import { Card, Button, CardImg, CardTitle, CardGroup,
  CardSubtitle, CardBody } from 'reactstrap';
 import { NavLink } from 'react-router-dom'
 
-export default function CatIndex({cats}) {
+export default function CatIndex({cars}) {
   return (
     <>
```
Comment (WalkerTexas-coder) [inline src/pages/CarIndex.js:8]: Card group can be the outer layer on this. Low priority but it will look nicer

File: src/pages/CarNew.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  const [name, setName ] = useState("")
+  const [year, setYear] = useState("")
+  const [enjoys, setEnjoys] = useState("")
+  const [image, setImage] = useState("")
+  // let [values, setValues] = useState({
+  //   name: null, 
+  //   year: null,
+  //   enjoys: null,
+  //   image: null
+  // })
```
Comment (WalkerTexas-coder) [inline src/pages/CarNew.js:16]: Delete the deprecated code.

**Review by WalkerTexas-coder — APPROVED** [review]
Lot of good work here couple of code cleanliness points.

File: src/pages/CarNew.js (hunk 2 above)
Comment (don-yoon) [reply src/pages/CarNew.js:16]: Will do

**Pull request merged**